features/testbot/planetary.feature

```diff
@@ -54,8 +54,8 @@ Feature: Distance calculation
             | ab    |
 
         When I route I should get
-            | from | to | route | distance       |
-            | a    | b  | ab,ab | 8905559m ~0.1% |
```

Comment:
Btw I quickly compared old and new values against Vincenty formula(I used https://www.npmjs.com/package/node-vincenty) and new values are more precise(I lost the exact values, but it is easy to repeat the experiment).